include/picongpu/particles/atomicPhysics/rateCalculation/BoundFreeFieldTransitionRates.hpp

```diff
@@ -39,77 +40,108 @@ namespace picongpu::particles::atomicPhysics::rateCalculation
     template<atomicPhysics::enums::ADKLaserPolarization T_ADKLaserPolarization>
     struct BoundFreeFieldTransitionRates
     {
-        /** rate for bound-free field ionization transition of ion depending on the local electric field
-         *
-         * @tparam T_ChargeStateDataBox instantiated type of dataBox
-         * @tparam T_AtomicStateDataBox instantiated type of dataBox
-         * @tparam T_BoundFreeTransitionDataBox instantiated type of dataBox
+    private:
+        struct ResultStruct
+        {
+            // unitless
+            PMACC_ALIGN(effectivePrincipalQuantumNumber, float_X);
+            // in elementary charge
+            PMACC_ALIGN(screenedCharge, float_X);
```

Comment:
Maybe you should just choose short names here `nEff` and `Z` such that the code using it down below doesn't need to unpack that.

Reply:
done